src/workloads/query/LimitSkip.yml

```diff
@@ -17,30 +17,30 @@ GlobalDefaults:
   Collection: &Collection Collection0
   DocumentCount: &DocumentCount 1e6
   Threads: &Threads 1
-  MaxPhases: &MaxPhases 6
+  MaxPhases: &MaxPhases 8
   LimitSkipNumber: &limitSkipNum {^Inc: {start: 1, step: 100}}
 
 ActorTemplates:
 - TemplateName: RunAggCommand
   Config:
-    Name: {^Parameter: {Name: "name", Default: "RunAggCommand"}}
+    Name: &Name {^Parameter: {Name: "name", Default: "RunAggCommand"}}
     Type: CrudActor
     Database: *Database
     Collection: *Collection
     Threads: 4
     Phases:
       OnlyActiveInPhases:
-        Active: {^Parameter: {Name: "active", Default: [0]}}
+        Active: {^Parameter: {Name: "active", Default: {unused: "Invalid phase number."}}}
         NopInPhasesUpTo: *MaxPhases
         PhaseConfig:
           Repeat: 1000
           Database: *Database
           Collection: *Collection
           Operations:
-          - OperationMetricsName: sbe
+          - OperationMetricsName: *Name
```

Comment:
This will happen automatically, so you don't need to specify this
https://github.com/mongodb/genny/blob/dcab85f33da584a2a2c7e2334c2499eb584627d5/src/cast_core/src/CrudActor.cpp#L1637

Reply:
Oh that's nice, done!